Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "vart"
 publish = true
-version = "0.4.1"
+version = "0.4.2"
```

Comment:
I think this requires a major bump (0.5.0). Semver check fails

```
--- failure inherent_method_missing: pub method removed or renamed ---

Description:
A publicly-visible method or associated fn is no longer available under its prior name. It may have been renamed or removed entirely.
        ref: https://doc.rust-lang.org/cargo/reference/semver.html#item-remove
       impl: https://github.com/obi1kenobi/cargo-semver-checks/tree/v0.35.0/src/lints/inherent_method_missing.ron

Failed in:
  Tree::bulk_insert, previously in file /Users/kfarhan/.cargo/registry/src/index.crates.io-6f17d22bba15001f/vart-0.4.1/src/art.rs:1001

--- failure struct_missing: pub struct removed or renamed ---

Description:
A publicly-visible struct cannot be imported by its prior path. A `pub use` may have been removed, or the struct itself may have been renamed or removed entirely.
        ref: https://doc.rust-lang.org/cargo/reference/semver.html#item-remove
       impl: https://github.com/obi1kenobi/cargo-semver-checks/tree/v0.35.0/src/lints/struct_missing.ron

Failed in:
  struct vart::art::KV, previously in file /Users/kfarhan/.cargo/registry/src/index.crates.io-6f17d22bba15001f/vart-0.4.1/src/art.rs:871
```